src/Facades/CRUDColumns.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Backpack\CRUD\Facades;
+
+use Illuminate\Support\Facades\Facade;
+
+class CRUDColumns extends Facade
+{
+    protected static function getFacadeAccessor()
```

Comment:
PHP doc?

Reply:
ALSO, while I'm on a no documentation run, I wonder how @tabacitu would plan to get the customer facing docs done for all these changes?